OtripleS.Web.Api.Tests.Unit/Services/CourseAttachments/CourseAttachmentServiceTests.Exceptions.RetrieveById.cs

```diff
@@ -0,0 +1,169 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using OtripleS.Web.Api.Models.CourseAttachments;
+using OtripleS.Web.Api.Models.CourseAttachments.Exceptions;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.CourseAttachments
+{
+    public partial class CourseAttachmentServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowDependencyExceptionOnRetrieveWhenSqlExceptionOccursAndLogItAsync()
+        {
+            // given
+            Guid someAttachmentId = Guid.NewGuid();
+            Guid someCourseId = Guid.NewGuid();
+            SqlException sqlException = GetSqlException();
+
+            var expectedCourseAttachmentDependencyException = 
+                new CourseAttachmentDependencyException(sqlException);
+
+            this.storageBrokerMock.Setup(broker =>
+                 broker.SelectCourseAttachmentByIdAsync(someCourseId, someAttachmentId))
```

Comment:
Why do we care about input parameters in the mock?
https://github.com/hassanhabib/The-Standard-Journal#itisanyt-in-exceptions-tests